custom_code/views.py

```diff
@@ -1463,6 +1472,382 @@ def load_lightcurve_view(request):
     return HttpResponse(json.dumps(context), content_type='application/json')
 
 
+def load_dash_lightcurve_view(request):
+    """Lazy-load the dash lightcurve plot for the overview tab"""
+    target_id = request.GET.get('target_id')
```

Comment:
Could you get around using the request.GET method everywhere by just adding a get decorator at the top of the function. @jchate6 Is that right?